### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Sarabjeet108**
update hyperlink

### CODE REVIEW

**Comment by ghost** [comment]
Thank you for the contribution, @Sarabjeet108 ! We just want to advise you that any contributions must fall under our license and cannot be restricted or compensated for in any way. If that is acceptable we will move forward with this pull request. 

My only feedback is that we would like `License` to stay uppercase; only the url needs to be changed for this issue.

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ When a new release (or version) occurs, change the Unreleased section of the [CH
 
 See also the list of [contributors](../../graphs/contributors) who participated in this project.
 
-## License
+## license
```
Comment (ghost) [inline README.md]: Leave uppercase

Comment (Sarabjeet108) [reply README.md]: Thank you for your feedback. I have changed 'license' back to uppercase.

Comment (ghost) [reply README.md]: Thank you!

**Pull request merged**